Fix:

error: use of undeclared identifier 'TCP_KEEPCNT'
error: use of undeclared identifier 'TCP_KEEPINTVL'

https://bugs.xdebug.org/1878
https://github.com/xdebug/xdebug/commit/2773ad5297a52605a283655e4f862e6547ca1854
--- src/debugger/com.c.orig	2020-09-28 05:23:47.000000000 -0500
+++ src/debugger/com.c	2023-12-17 04:16:13.000000000 -0600
@@ -111,17 +111,21 @@
 		return;
 	}
 
+# if defined(TCP_KEEPCNT)
 	optval = 20;
 	ret = setsockopt(fd, SOL_TCP, TCP_KEEPCNT, &optval, optlen);
 	if (ret) {
 		return;
 	}
+# endif
 
+# if defined(TCP_KEEPINTVL)
 	optval = 60;
 	ret = setsockopt(fd, SOL_TCP, TCP_KEEPINTVL, &optval, optlen);
 	if (ret) {
 		return;
 	}
+# endif
 }
 #endif
 
